examples/specs/window_sum_waterfall_chart.vl.json

```diff
@@ -0,0 +1,104 @@
+{
+  "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
+  "data": {
+    "values": [
+      {"label": "Start", "amount": 4000},
+      {"label": "Jan", "amount": 1707},
+      {"label": "Feb", "amount": -1425},
+      {"label": "Mar", "amount": -1030},
+      {"label": "Apr", "amount": 1812},
+      {"label": "May", "amount": -1067},
+      {"label": "Jun", "amount": -1481},
+      {"label": "Jul", "amount": 1228},
+      {"label": "Aug", "amount": 1176},
+      {"label": "Sep", "amount": 1146},
+      {"label": "Oct", "amount": 1205},
+      {"label": "Nov", "amount": -1388},
+      {"label": "Dec", "amount": 1492},
+      {"label": "End", "amount": 0}
+    ]
+  },
+  "width": 700,
+  "height": 450,
+  "transform": [
+    {"window": [{"op": "row_number", "as": "row"}]},
```

Comment:
Do we really need to run window twice here?  (Wouldn't the default sort order be by row number?)